web/src/components/vis/VisTile.vue

```diff
@@ -5,6 +5,10 @@ export default {
       type: String,
       default: '',
     },
+    loading: {
+      default: false,
+      validator: () => true,
```

Comment:
Doesn't seem like we need a validator here.

Reply:
You have to have either a validator or a type to pass the linter.   This is the only way I know to accept any value without warnings, and everything in javascript has truthiness.

Reply:
Maybe a comment explaining why we have an all-pass validator here would be good.

Reply:
Wait, why not just say this is a boolean value? Surely that's better for code style throughout the uses of this `loading` prop than allowing any truthy/falsy value?